src/quo2/components/selectors/disclaimer.cljs

```diff
@@ -0,0 +1,29 @@
+(ns quo2.components.selectors.disclaimer
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.selectors.selectors :as selectors]))
+
+(defn disclaimer [{:keys [checked? on-change accessibility-label]} label]
+  [rn/view {:style
+            {:display :flex
+             :flex 2
+             :flex-direction :row
+             :background-color
+             (colors/theme-colors
+              colors/neutral-5
+              colors/neutral-80-opa-40)
+             :padding 11
+             :align-self :stretch
+             :border-radius 12
+             :border-width 1
+             :border-color (colors/theme-colors
+                            colors/neutral-20
+                            colors/neutral-70)}}
```

Comment:
hi @J-Son89, Thank you for PR
1. [display](https://reactnative.dev/docs/layout-props#display) property is redundant as this is the default value.
2. why `:flex 2`?
3. Please indent map values

Reply:
yep, they're just pieces I should have cleaned up. Will adjust and indent :)